.github/workflows/build_documentation.yml

```diff
@@ -50,6 +50,42 @@ jobs:
         run: |
           set -ex
 
+          # A list of all docs that will be build
+          #
+          # Install 24.05 docs under 'stable'
+          # TODO: consider having `<release>` instead of `stable`?
+          versions='
+            [
+              {
+                "branch": "main",
+                "nixpkgsBranch": "nixos-unstable",
+                "subPath": null
+              },
+              {
+                "branch": "nixos-24.05",
+                "nixpkgsBranch": "nixos-24.05",
+                "subPath": "stable"
+              }
+            ]
+          '
+
+          # Converts the "versions" json into an "others" json, for use in the drv overrides
+          #
+          # 1: current branch - the matching entry will have url set to null
+          # Output is written stdout
+          othersJson() {
+            echo "$versions" | jq \
+             --arg current "$1" \
+             --arg urlpfx "https://nix-community.github.io/$repoName" \
+              'map({
+                branch: .branch,
+                nixpkgsBranch: .nixpkgsBranch,
+                url: (if .branch == $current then null else (
+                  [ $urlpfx, .subPath, "" ] | map(select(.)) | join("/")
+                ) end)
```

Comment:
We could probably just pass in a `baseHref` instead of a full URL;

An absolute path link should work just fine, even without the domain (etc).

We'd also then be able to check "is current" within the actual derivation, by comparing `baseHref == elem.baseHref` (instead of a null check).

That way _this_ step could be much simpler, perhaps totally unnecessary.

Reply:
Done